### DISCUSSION TITLE
[FIXED] Gift card balance add taxes

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#9950] [General] [answered]
Hi everyone,

we're facing this issue regarding gift cards:
basically if we buy a gift card, let say with a value of $50, 
we get a balance of 53.75$, that's the price of the card PLUS taxes 
![Alé Bike Wear 8-6-2022 11-58-43](https://user-images.githubusercontent.com/31857092/172594090-599c351e-8404-4055-8ffe-0c8e15807860.png)

![-34- Libero Mail - Posta 8-6-2022 11-50-13](https://user-images.githubusercontent.com/31857092/172587577-22c1bf6a-3e67-44b3-ac21-1924bb3aeddf.png)

And whenever we're trying to use it, we've found 53.75USD instead of 50USD
(see also checkout giftCards obj attached below)

 `[
    {
        "initialBalance": {
            "amount": 53.75,
            "currency": "USD",
            "__typename": "Money"
        },
        "currentBalance": {
            "amount": 53.75,
            "currency": "USD",
            "__typename": "Money"
        },
        "displayCode": "4BFB",
        "product": {
            "slug": "gift-card-50",
            "__typename": "Product"
        },
        "__typename": "GiftCard"
    }
]`

Do you have any clue/hint on this behaviour?
Please let us know about it, thanks, it's a problem because customers are still using a gift card with an higher amount than expected.

Thanks in advance, 
Lorenzo

**Comment by karolkielecki** [comment]
Hi @Anubiella, which version are you running? Is that Saleor Cloud or you're using Saleor locally?

**Reply by Anubiella** [reply]
locally, for the version
![Configuration - Ale Bike Wear 14-6-2022 09-10-03](https://user-images.githubusercontent.com/31857092/173515765-475cc4a6-4da1-469f-82d6-dbb300ba345c.png)

**Comment by patrys** [comment, marked as answer]
@Anubiella Hi, is there a reason you want to charge taxes on gift card sales? Given that gift cards are a payment method, you'd effectively charge double taxation on the actual goods.

**Reply by Anubiella** [reply]
Hi, I'll check this asap and let you know, thanks a lot!

**Comment by Anubiella** [comment]
Fine!

It was just the taxes charging that caused the strange balance.

Thanks a lot,
god bless

Lorenzo
